package.json

```diff
@@ -49,5 +50,6 @@
     "napi_versions": [
       4
     ]
-  }
-}
+  },
+  "zstd_version": "1.5.6"
```

Comment:
It's highly improbable that this would ever clash with existing tooling but to be beyond certain that this is our field for us to manage can we "namespace" it with a prefix like we have in [mongodb-client-encryption](https://github.com/mongodb-js/mongodb-client-encryption/blob/main/package.json#L38) and [bson](https://github.com/mongodb/js-bson/blob/main/package.json#L91)